providers/implementations/asymciphers/rsa_enc.c.in

```diff
@@ -172,9 +172,9 @@ static int rsa_encrypt(void *vprsactx, unsigned char *out, size_t *outlen,
     }
 #endif
 
-    if (out == NULL) {
-        size_t len = RSA_size(prsactx->rsa);
+    size_t len = RSA_size(prsactx->rsa);
```

Comment:
Move the declaration up to line 158.  The code style says to not intermingle declaration and code.